### DISCUSSION TITLE
Contact point for GreenSOftware Foundation for a presentation

### DISCUSSION THREAD

**Discussion by shoareau** [Green-Software-Foundation/discussions#76] [Community Working Group]
Good morning ,
I want to contact some one from the GreenSoftware foundation.
My goal is present this topic to the embedded software community of the company i worked in, in order to bring awareness and to contribue.

Who can i contact?

Thank you

Stephane Hoareau

**Comment by NAMRATA-WOKE** [comment]
Hi Stephane, please reach out to Namrata Narayan, our Director of Communications & Member Relations at namrata@greensoftware.foundation.
